src/go/rpk/pkg/cli/cmd/redpanda/tune/help.go

```diff
@@ -35,7 +35,7 @@ func NewHelpCommand() *cobra.Command {
 	return &cobra.Command{
 		Use:   "help <tuner>",
```

Comment:
The more I look at this function, and given my comment below re: the long help text in `rpk redpanda tune`, I kinda think this function should be improved by having much more help text here as to what each tuner is doing.

Not something to do in this PR, but I think we should create an issue to expand the help text here with

* What the tuner is doing
* Why it is doing it
* potentially, how to revert what the tuner did